style.css

```diff
@@ -0,0 +1,247 @@
+
+html {
```

Comment:
- [ ] There is a white space on the top and on the left of your website, kindly fix this error.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/185494823-c409b5ea-11e1-436f-b49c-91a114c8df74.png)